person.rb

```diff
@@ -1,21 +1,23 @@
-class Person
-  attr_reader :id
-  attr_accessor :name, :age
+require './nameable'
 
-  def initilialize(age, name: 'Unknown', parent_permission: true)
-    @id = Random.rand(1..100)
+class Person < Nameable
+  attr_accessor :id, :name, :age
+
+  def initialize(id, age, name: 'Unknown', parent_permission: 'Unknown')
+    @id = id
     @name = name
     @age = age
     @parent_permission = parent_permission
+    super
   end
 
   def can_use_services?
-    of_age? || @parent_permission
+    return true if of_age? || @parent_permission == true
   end
 
   private
 
   def of_age?
-    @age >= 18
+    return true if @age >= 18
   end
 end
```

Comment:
**NOTTICE:** It would be good once you have implemented all the above changes to run all the below commands in order to check if your app is working as expected before requesting a new review:
```ruby
 person = Person.new(22, 'maximilianus')
 person.correct_name
 capitalizedPerson = CapitalizeDecorator.new(person)
 capitalizedPerson.correct_name
 capitalizedTrimmedPerson = TrimmerDecorator.new(capitalizedPerson)
 capitalizedTrimmedPerson.correct_name
```